tap_jira/streams.py

```diff
@@ -110,7 +147,12 @@ def write_page(self, page):
         extraction_time = singer.utils.now()
         for rec in page:
             with Transformer() as transformer:
-                rec = transformer.transform(rec, stream.schema.to_dict(), stream_metadata)
+                try:
+                    rec = transformer.transform(rec, stream.schema.to_dict(), stream_metadata)
+                except SchemaMismatch as ex:
+                    # Checking if schema-mismatch is occurring for datetime value
```

Comment:
Provide a card and detailed description here in a comment.

Reply:
Added the card description with the card link